src/main/java/com/minecolonies/coremod/research/LocalResearch.java

```diff
@@ -57,11 +59,14 @@ public boolean research(final IResearchEffectManager effects, final ILocalResear
     {
         if (state == ResearchState.IN_PROGRESS)
         {
-            progress++;
+            progress += (1 * MinecoloniesAPIProxy.getInstance().getConfig().getServer().researchSpeedMultiplier.get());
             if (progress >= BASE_RESEARCH_TIME * Math.pow(2, depth - 1))
```

Comment:
I don't think I want this to be configurable via the config file so easily. Either people play creative mode and get it instant, or wait 1h for a level 1 research.

Reply:
Largely trying to consider the modpack case, where there may be a whole ton of extra recipe-related use cases (and testing non-creative behavior).   Would making it a separate Json in the data pack avoid the ‘every schmuck immediately configs to max’ issue?

Reply:
Potentially we could have a setting per "main branch" on how long each research needs in the datapack.

Like:

"combat": "1: (1h per level)
"civilian": 2 (2h per level)
"technology": 0.5 (30min per level)

Could be a configuration, this way we can also display the exact time each of them needs in the top

Reply:
Prefer to keep the power-of-two approach in that case over defining levels individually?

Reply:
I'd keep the power of two. If one day a pack maker actually wants to define this more fine grained, we can think about this.